profiles/nextstrain-public.yaml

```
@@ -173,9 +190,12 @@ array-builds:
         - 2Y
       reference_min_date:
         - 8Y
+      resolution:
+        - 2y
     build_name: "flu_seasonal_h3n2_2y"
     build_params:
       lineage: h3n2
+      auspice_name: "flu_seasonal_{lineage}_{{segment}}_2y"
```

Should this also be using the `resolution` wildcard? 

```suggestion
      auspice_name: "flu_seasonal_{lineage}_{{segment}}_{resolution}"
```

---

The 2y H3N2 build is a special case with its own build params, so this would technically work, but it's clearer to just always use the resolution variable throughout. Fixed in 0a4db33.